app/src/lib/git/cherry-pick.ts

```diff
@@ -80,6 +184,111 @@ function parseCherryPickResult(result: IGitResult): CherryPickResult {
   }
 }
 
+/**
+ * Inspect the `.git/sequencer` folder and convert the current cherry pick
+ * state into am `ICherryPickProgress` instance as well as return an array of
+ * remaining commits queued for cherry picking.
+ *  - Progress instance required to display progress to user.
+ *  - Commits required to track progress after a conflict has been resolved.
+ *
+ * This is required when Desktop is not responsible for initiating the cherry
+ * pick and when continuing a cherry pick after conflicts are resolved:
+ *
+ * It returns null if it cannot parse an ongoing cherry pick. This happens when,
+ *  - There isn't a cherry pick in progress (expected null outcome).
+ *  - Runs into errors parsing cherry pick files. This is expected if cherry
+ *    pick is aborted or finished during parsing. It could also occur if cherry
+ *    pick sequencer files are corrupted.
+ */
+export async function getCherryPickSnapShot(
```

Comment:
Nit: for rebase, it's spelled like `getRebaseSnapshot`
```suggestion
export async function getCherryPickSnapshot(
```